pkg/base1/cockpit.css

```diff
@@ -665,6 +665,7 @@ tr.listing-item .listing-toggle {
     padding: 0 !important;
     width: 35px;
     color: #000000;
+    cursor: pointer;
```

Comment:
@stefwalter @larsu I think this is a valid addition to the base stylesheet: the toggle element should always be clickable